src/utils/foundry.test.ts

```diff
@@ -0,0 +1,12 @@
+import { checkIsFoundryInstalled } from '@utils/foundry'
+import { describe, expect, test } from 'vitest'
+
+describe('foundry', () => {
+  describe('checkIsFoundryInstalled', () => {
+    test('should return true if foundry is installed', async () => {
+      const result = await checkIsFoundryInstalled()
+
+      expect(result).toBeTruthy()
+    })
+  })
+})
```

Comment:
The test should have the same result on every computer. Should we mock the implementation of the `spawnChild` instead? We can mock that if it does not throw error, this function should return true. If it does return the error, this function should throw the error instead?